LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDevice.cs

```diff
@@ -276,8 +276,8 @@ public virtual async Task<bool> InitializeAsync(NetworkServerConfiguration confi
                 // ABP Case
                 try
                 {
-                    AppSKey = desiredTwin.ReadRequiredString(TwinProperty.AppSKey);
-                    NwkSKey = desiredTwin.ReadRequiredString(TwinProperty.NwkSKey);
+                    AppSKey = desiredTwin.TryRead(TwinProperty.AppSKey, out AppSessionKey someAppSessionKey) ? someAppSessionKey : throw new InvalidLoRaDeviceException($"'{TwinProperty.AppSKey}' is empty");
```

Comment:
nit: should we add a ReadRequiredProperty instead?

Reply:
Done in 6f3c2197acaa8a6f83a14fe9e4b34e7056b33bf4